### DISCUSSION TITLE
Compile error after update - There is no argument given that corresponds to the required parameter 'skiaModel' of 'SKSvg.ToPicture(SvgFragment, SkiaModel, IAssetLoader)'

### DISCUSSION THREAD

**Discussion by Jarek300** [wieslawsoltes/Svg.Skia#182] [Q&A]
The code is:
```
            SvgDocument? svgDocument = SvgExtensions.Open(stream);
            if (svgDocument == null)
                return;
...
            using SKPicture? picture = SKSvg.ToPicture(svgDocument);

```
Compiler error

There is no argument given that corresponds to the required parameter 'skiaModel' of 'SKSvg.**ToPicture**(SvgFragment, SkiaModel, IAssetLoader)'.

What SkiaModel and what  IAssetLoader should I use as the parameters? Previously ToPicture didn't have these parameters.

**Comment by wieslawsoltes** [comment]
https://github.com/wieslawsoltes/Svg.Skia/blob/5a85fa83158047f4d746362adfb65c4e2d7f6d77/src/Svg.Skia/SKSvg.NoModel.cs#L32

**Comment by Jarek300** [comment]
What does it mean? 

The line 

Picture = SvgExtensions.ToPicture(svgDocument, AssetLoader, out var _, out _); 

does not compile.

![image](https://github.com/wieslawsoltes/Svg.Skia/assets/10242002/0f7c6c6f-0d81-448d-98cb-2883d470f8b7)

SvgExtensions does not have a method ToPicture. Accroding to Visual Studio and its disassembler:

#region Assembly Svg.Model, Version=1.0.0.3, Culture=neutral, PublicKeyToken=dafe96fe6c845a74
